src/components/layout/nav.js

```diff
@@ -599,48 +599,58 @@ const StyledLogo = styled(LogoLink)`
         }
     }
 `
+
+const NavInterimContainer = styled.div`
+    position: relative;
+`
+
 export const NavInterim = ({ interim_type }) => (
     <InterimNav>
         <CFDWarning />
-        <Container jc="space-between" p="2.4rem 0">
-            <Flex ai="center" jc="flex-start">
-                <Show.Desktop>
-                    <StyledLogo to={`/interim/${interim_type}`} aria-label={localize('Home')}>
-                        <Flex ai="center">
-                            <img src={Logo} alt="logo" width="190px" height="27px" />
-                            <img
-                                src={LogoCombinedShape}
-                                alt="logo combined shape desktop"
-                                width="120"
-                                height="17"
-                            />
-                        </Flex>
-                    </StyledLogo>
-                </Show.Desktop>
-                <Show.Mobile>
-                    <LogoLinkMobile to={`/interim/${interim_type}`} aria-label={localize('Home')}>
-                        <Flex>
-                            <img src={LogoOnly} alt="logo only 2" width="115px" height="27px" />
-                            <LogoDescription ai="center">
-                                <Line />
+        <NavInterimContainer>
+            <Container jc="space-between" p="2.4rem 0">
+                <Flex ai="center" jc="flex-start">
+                    <Show.Desktop>
+                        <StyledLogo to={`/interim/${interim_type}`} aria-label={localize('Home')}>
+                            <Flex ai="center">
+                                <img src={Logo} alt="logo" width="190px" height="27px" />
```

Comment:
we can omit `px` from `width` and `height` of `img` tag

Reply:
done.